components/tensorflow-notebook-image/releaser/components/workflows.libsonnet

```diff
@@ -82,6 +82,9 @@
       // command: List to pass as the container command.
       local buildTemplate(step_name, command, env_vars=[], sidecars=[]) = {
         name: step_name,
+        retryStrategy: {
```

Comment:
Is it flaky? Can you add some comment on why we need retry

Reply:
Done